iOS/Flows/Conversation/ConversationViewController.swift

```diff
@@ -307,9 +308,21 @@ class ConversationViewController: FullScreenViewController,
             self.sendMessageOverlay.alpha = 1
         }
 
-        self.sendMessageOverlay.size = CGSize(width: self.collectionView.width * 0.8,
-                                              height: self.collectionView.height * 0.27)
-        self.sendMessageOverlay.match(.top, to: .top, of: self.collectionView)
+        // Show the send message overlay so the user can see where to drag the message
+        if let centerCell = self.collectionView.getCentermostVisibleCell() as? MessageCell,
+           let messageSubcell = centerCell.getFrontmostMessageCell() {
+
+            // If possible put the message overlay around front most message
+            var overlayFrame = messageSubcell.convert(messageSubcell.bounds, to: self.contentContainer)
+            overlayFrame.top += Theme.contentOffset
+            self.sendMessageOverlay.frame = overlayFrame
```

Comment:
Converts the frame of the cell into the conversation view controller coordinate space.